compiler/rustc_typeck/src/check/fn_ctxt/checks.rs

```diff
@@ -226,6 +227,20 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                     String::from("()"),
                     Applicability::MachineApplicable,
                 );
+            } else if sugg_tuple_wrap {
+                let (first, last) = match args {
+                    [first, .., last] => (first, last),
+                    _ => bug!("shouldn't be called without 2-or-more-tuple"),
```

Comment:
What happens with the following code? Can you add tests?

```rust
let x: Option<()> = Some();
let x: Option<(i32,)> = Some(1);
```

Reply:
Empty/unit tuple is suggested already ([see `sugg_unit`](https://github.com/rust-lang/rust/blob/8b09ba6a5d5c644fe0f1c27c7f9c80b334241707/compiler/rustc_typeck/src/check/fn_ctxt/checks.rs#L225)). For the non-empty, but missing-tuple case, I've modified code that suggests changing `(1)` to `(1,)` to suggest this less conditionally, although I'd like to see whether it would be appropriate to refactor so the 1-tuple case isn't special-cased in this way.